### ISSUE TITLE
CircleCI publishes outdated `plot-schema.json`

### ISSUE THREAD

**Issue by antoinerg** [plotly/plotly.js#3159]
On CircleCI `publish` happens right after the `build` job.

The build job on the CI is `npm run cibuild` which doesn't update the `plot-schema.json`. We should update it either by:
1. adding code to https://github.com/plotly/plotly.js/blob/master/tasks/cibundle.js 
2. creating a new task only for that purpose which would only be executed in the `publish` job.

Option 1 adds 15 seconds to the build step. So I would favor option 2 since it can be runned in parallel.

### master
``` 
[antoine@newton:~/plotly/plotly.js]$ time npm run cibuild

> plotly.js@1.41.3 cibuild /home/antoine/plotly/plotly.js
> npm run preprocess && node tasks/cibundle.js

###
> plotly.js@1.41.3 preprocess /home/antoine/plotly/plotly.js
> node tasks/preprocess.js

ok plotly-geo-assets.js
ok plotly.js
ok plotly.min.js

real    0m39.960s
user    0m59.195s
sys     0m2.339s
```
### Option 1
```
[antoine@newton:~/plotly/plotly.js]$ time npm run cibuild

> plotly.js@1.41.3 cibuild /home/antoine/plotly/plotly.js
> npm run preprocess && node tasks/cibundle.js


> plotly.js@1.41.3 preprocess /home/antoine/plotly/plotly.js
> node tasks/preprocess.js

ok plotly-geo-assets.js
ok plotly-with-meta.js
ok plot-schema.json
ok plotly.js
ok plotly.min.js

real    0m53.048s
user    1m21.699s
sys     0m3.376s
```

**Comment by etpinard** [comment]
Voting for just calling 

https://github.com/plotly/plotly.js/blob/fdcdd227ea4ef2380bbf25682049f506be1319ec/tasks/bundle.js#L64-L74

in the publish step.

**Comment by antoinerg** [comment]
Sounds like a very good idea! I will remove the PR https://github.com/plotly/plotly.js/pull/3160 I just made!
